### DISCUSSION TITLE
Why do you use locize vs better/other Solutions

### DISCUSSION THREAD

**Discussion by jannesblobel** [i18next/next-i18next#2232] [Q&A] [answered]
Hey guys, 
First of all, sorry if I'm asking so directly. I don't want to be rude. 
But I would love to know why you recommend locize for an open-source project/use it yourself. I couldn't contribute another language with the editor ( or I have overlooked it). 
I believe there are better solutions, especially for dev's first communities like i18next

**Comment by adrai** [comment, marked as answer]
Because without locize sponsoring the i18next work, i18next would not survive: https://locize.com/blog/maintaining-open-source/
The creators of i18next are also the founders of locize.
Adding additional languages to locize should be straight forward: https://docs.locize.com/integration/getting-started/add-content#add-languages
If you need more help or have other suggestions on how we could improve it, please write to support@locize.com
